app/models/manageiq/providers/redhat/discovery.rb

```diff
@@ -18,6 +17,8 @@ def probe(ost)
           private
 
           def ovirt_exists?(host, logger = nil)
+            require 'ovirtsdk4'
```

Comment:
Similarly we can probably emebded the network discovery gem (in another PR of course)